### ISSUE TITLE
What if using a resnet-152 when extracting features instead of the Faster-RCNN?

### ISSUE THREAD

**Issue by Kyubyong** [krasserm/fairseq-image-captioning#17]
Hi, I wonder if we can use the extracted features from the resnet-152 model, not from the Faster-RCNN because the former is easy to implement.

**Comment by Kyubyong** [comment]
Oh, never mind. Now I see there is an option for InceptionNet. Then was the pretrained models (checkpoint 20, 24) trained using the --feature`grid' option? Or the `obj` option?

**Comment by krasserm** [comment]
They were trained as described in https://github.com/krasserm/fairseq-image-captioning/blob/master/README.md#training

**Comment by Kyubyong** [comment]
Thanks for your confirmation. Have you checked the performance of the pretrained model provided in https://github.com/krasserm/fairseq-image-captioning/tree/wip-train-inception? I'm curious how good the grid based model is compared to the obj. based model.

**Comment by krasserm** [comment]
The object-based model is significantly better but when I trained the grid-based model long time ago I didn't really tune hyper-parameters. So it may be worth re-training it with hyper-parameters similar to object-based training (lr, warmup, ...), at least as starting point. On the other hand, most image captioning papers report object-based approaches to be superior to grid-based approaches.
